src/Form/Type/VichFileType.php

```diff
@@ -26,35 +25,14 @@
  */
 class VichFileType extends AbstractType
 {
-    /**
-     * @var StorageInterface
-     */
-    protected $storage;
-
-    /**
-     * @var UploadHandler
-     */
-    protected $handler;
-
-    /**
-     * @var PropertyMappingFactory
-     */
-    protected $factory;
-
-    /**
-     * @var PropertyAccessorInterface
-     */
-    protected $propertyAccessor;
+    protected PropertyAccessorInterface $propertyAccessor;
 
     public function __construct(
-        StorageInterface $storage,
-        UploadHandler $handler,
-        PropertyMappingFactory $factory,
+        protected StorageInterface $storage,
+        protected UploadHandler $handler,
+        protected PropertyMappingFactory $factory,
```

Comment:
I would guess most of the constructor-injected services across all files could also be marked as `readonly`?

Reply:
You're right. I hope to have caught every occurence